diff --git a/modules/bat_event/src/Entity/Form/EventForm.php b/modules/bat_event/src/Entity/Form/EventForm.php
--- a/modules/bat_event/src/Entity/Form/EventForm.php
+++ b/modules/bat_event/src/Entity/Form/EventForm.php
@@ -189,16 +189,6 @@
       }
     }
 
-    $form['event_dates']['widget'][0]['value']['#date_timezone'] = 'UTC';
-    $form['event_dates']['widget'][0]['end_value']['#date_timezone'] = 'UTC';
-
-    if (isset($form['event_dates']['widget'][0]['value']['#default_value'])) {
-      $form['event_dates']['widget'][0]['value']['#default_value']->setTimezone(new \DateTimeZone('UTC'));
-    }
-    if (isset($form['event_dates']['widget'][0]['end_value']['#default_value'])) {
-      $form['event_dates']['widget'][0]['end_value']['#default_value']->setTimezone(new \DateTimeZone('UTC'));
-    }
-
     if ($this->request->query->get(MainContentViewSubscriber::WRAPPER_FORMAT) == 'drupal_ajax') {
       $form['actions']['submit']['#attributes']['class'][] = 'use-ajax-submit';
     }
